common/meta/raft-store/src/sled_key_spaces.rs

```diff
@@ -109,6 +109,15 @@ impl SledKeySpace for ClientLastResps {
     type V = ClientLastRespValue;
 }
 
+/// Key-Value types for meta service addrs in sled::Tree:
+pub struct MetaSrvAddrs {}
+impl SledKeySpace for MetaSrvAddrs {
+    const PREFIX: u8 = 11;
+    const NAME: &'static str = "metasrv-addr";
+    type K = String;
+    type V = String;
+}
+
 /// Enum of key-value pair types of all key spaces.
```

Comment:
Storing the client-grpc address in the existing keyspace `Nodes` would be better.

Next PR!